03week/arrayHW.js

```diff
@@ -0,0 +1,41 @@
+let cars = ["Ford", "Mustang", "Chevy", "Saturn"];
+
+console.log(cars.length);
+
+let moreCars = ["Acura", "BMW", "Audi", "Honda"];
+
+let totalCars = cars.concat(moreCars);
+
+console.log(totalCars.indexOf("Honda"));
+
+console.log(totalCars.lastIndexOf("Ford"));
+
+let stringOfCars = totalCars.join();
+
+totalCars = stringOfCars.split(',');
+
+let carsInReverse = totalCars.reverse();
+
+carsInReverse.sort();
+
+console.log(carsInReverse);
+
+alert(carsInReverse.indexOf('Acura'));
+
+carsInReverse.splice(1, 2, 'Ford', 'Honda');
```

Comment:
Missing number 9, and using slice, make sure you know the difference between splice and slice.